pkg/api/v20191231preview/openshiftclusterslistorgetkubernetesobjects.go

```diff
@@ -0,0 +1,31 @@
+package v20191231preview
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+type OpenShiftClusterListOrGetKubernetesObject struct {
+	Name            string            `json:"name,omitempty"`
+	GenerateName    string            `json:"generateName,omitempty"`
+	Namespace       string            `json:"namespace,omitempty"`
+	UID             string            `json:"uid,omitempty"`
+	ResourceVersion string            `json:"resourceVersion,omitempty"`
+	Labels          map[string]string `json:"labels,omitempty"`
+	Annotations     map[string]string `json:"annotations,omitempty"`
+	ClusterName     string            `json:"clusterName,omitempty"`
+}
+
+// OpenShiftClusterListOrGetKubernetesObjects represents one or more kubernetes objects
+type OpenShiftClusterListOrGetKubernetesObjects struct {
```

Comment:
I am not using this in the golang client in hack/ocget/ocget.go and it won't get used in the geneva action(.net client).